src/viam/app/app_client.py

```diff
@@ -1581,25 +1630,33 @@ async def delete_robot(self, robot_id: str) -> None:
         request = DeleteRobotRequest(id=robot_id)
         await self._app_client.DeleteRobot(request, metadata=self._metadata)
 
-    async def list_fragments(self, org_id: str, show_public: bool = True) -> List[Fragment]:
+    async def list_fragments(
+        self, org_id: str, show_public: bool = True, visibilities: Optional[List[Fragment.Visibility]] = None
+    ) -> List[Fragment]:
         """Get a list of fragments under the currently authed-to organization.
 
         ::
 
-            fragments_list = await cloud.list_fragments(org_id="org-id", show_public=False)
+            fragments_list = await cloud.list_fragments(org_id="org-id", visibilities=[])
 
         Args:
             org_id (str): The ID of the organization to list fragments for.
                 You can obtain your organization ID from the Viam app's organization settings page.
-            show_public: Optional boolean specifying whether or not to only show public fragments. If True, only public fragments will
-                return. If False, only private fragments will return. Defaults to True.
+            visibilities: List of FragmentVisibilities specifying which types of fragments to include in the results.
+                If empty, by default only public fragments will be returned.
+            show_public: Deprecated: use visibilities instead. Optional boolean specifying whether or not to only show public
```

Comment:
Also, add the following below the param doc:

```python3
show_public (bool): **Deprecated**: Use ``visibilities`` instead
    Optional boolean specifying...

    .. deprecated:: 0.25.0
        Use ``visibilities`` instead.
```

Be sure to include the new line and tabs. This will add a deprecated line to the rendered documentation
<img width="522" alt="image" src="https://github.com/viamrobotics/viam-python-sdk/assets/2429728/ee5c5290-da11-4b3d-8015-dc514712e078">

ALSO: please add type hints to these doc strings

Reply:
You can check the rendering by running `make documentation` from the project root directory, and then opening the generated HTML and navigating to the function: `open docs/_build/html/autoapi/viam/app/app_client/index.html`

Reply:
this is the current output I'm getting with the changes!!
<img width="742" alt="image" src="https://github.com/viamrobotics/viam-python-sdk/assets/51691446/48da6269-f74d-4ce9-b021-64732f5da833">